ibc_data_schema.json

```diff
@@ -80,7 +80,43 @@
         },
         "additionalProperties": false
       }
-    }
+    },
+    "operators": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "chain_1",
+          "chain_2"
+        ],
+        "properties": {
+          "chain_1": {
+            "type": "object",
+            "$ref": "#/$defs/operator_info"
+          },
+          "chain_2": {
+            "type": "object",
+            "$ref": "#/$defs/operator_info"
+          },
+          "description": {
+            "type": "string",
+            "description": "Relayer addresses and metadata."
```

Comment:
`description` property should contain `Relayer addresses and metadata`? Can you give an example value?

Reply:
on second thought we probably do not need descriptions here as it will be redundant